src/plugins/discover/public/plugin.tsx

```diff
@@ -320,10 +320,9 @@ export class DiscoverPlugin
         // make sure the index pattern list is up to date
         await depsStart.data.indexPatterns.clearCache();
 
-        const { renderApp } = await import('./application/application');
-        const unmount = await renderApp('discover', params.element);
+        const { renderApp } = await import('./application');
+        const unmount = await renderApp(params.element);
```

Comment:
This `await` should no longer be needed.

Reply:
indeed, changed that